modules/ballerina-test/src/test/resources/test-src/types/map/map-value.bal

```diff
@@ -55,6 +57,16 @@ function testMapWithAnyFunctionInvocations() (string){
     return value;
 }
 
+function testMapOrder() (map)
+{
```

Comment:
Be consistent with the styling, the opening bracket should come in the same line. Fix in all the places.